vuepress/docs/next/tutorials/devops/installation/kubernetes/kubernetes-install.md

```diff
@@ -0,0 +1,47 @@
+# Install Entando on Kubernetes
+
+## Prerequisites
+
+### Infrastructure Requirements
+
+To install Entando directly on a Kubernetes (K8s) cluster, these are the minimum infrastructure requirements:
+
+- A working DNS server configured to resolve internal and external names. If you choose to configure K8s with a global wildcard domain name for the entire cluster, be sure it points to the public IP address of the external Load Balancer (LB) of the worker nodes.
+- A default [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is needed for the Entando operator to create the correct Persistent Volume Claims (PVC) for each pod.
+- Nginx [Ingress-Controller](https://kubernetes.github.io/ingress-nginx/deploy/) is needed to expose the Entando applications.
+
+### Minimum Hardware Requirements
+
+The minimum hardware requirements depends on how you intend to use Entando 6. The more functionality you need such as a content management system (CMS), the more you will need.
+#### For Entando install only:
+
+- 6 vCPU or virtual central processing unit  (6000 millicores)
+- 6GB of RAM
+- 5GB of allocated Storage distributed in many PVCs
+
+Configure a single worker node with the requirements above or designate 3 worker nodes with the minimum of 2VPC and 4GB of RAM each.
+
+#### For Entando + Microservices:
+
+- 500 millicores per Microservice or 
+- 1000 millicores per Microservice with a DB
+
+The instance tested was generated with a JHipster Entando blueprint.
+
+#### For Entando with CMS: 
+
+To install Entando with CMS, the hardware requirements will depend on the complexity and number of pages and contents. The minimum requirements are:
+
+- 9vCPU for a single CMS instance. To scale up, you need 4vCPU per replica.
+- 8GB of RAM for a single CMS instance. To scale up,  you need 4GB of RAM per replica.
+- A minumum of 50GB of storage for all the static resources generated by the use of Entando CMS.
+
+### Backup Strategies
+
+Using the managed DB services that automate backups and restores on a Kubernetes cluster like Azure, Red Hat OpenShift, Google K8s Service, Tanzu K8s Grid or Amazon Elastic is the recommended method to secure your content. If this is not an option, there are tools like [Velero](https://velero.io/) and [Cloudcase](https://cloudcasa.io/) to help with that. **In production environments, you should not use DBs running inside pods**.
```

Comment:
-> "automate and restore backups"
second person again
resolve what Velero does to help?

Reply:
1. I think they are services and maybe should even be capitalized. At the least I think they are standard processes.
2. to discuss
3. I think it's implied, since it's discussing backup strategies, they are tools for that.